genienlp/validate.py

```diff
@@ -410,7 +410,7 @@ def validate(task, val_iter, model, numericalizer, args, num_print=10):
             output.predictions, output.answers, metrics_to_return, args.reduce_metrics, model.tgt_lang
         )
 
-        results = {'beam search': output.predictions, 'answer': output.answers, 'context': output.contexts}
+        results = {'model prediction': output.predictions, 'answer': output.answers, 'context': output.contexts}
```

Comment:
Also to make printed results align nicely in terminal, we use the maximum number of characters among all keys: https://github.com/stanford-oval/genienlp/blob/1d3f64bfa4d5077c5fe2748977d88ccdc16dd548/genienlp/validate.py#L392 
Maybe it's time to change the "11" to `max(len(key) for key in results)`